error_113.go

```diff
@@ -0,0 +1,22 @@
+// +build go1.13
```

Comment:
Did I understand correctly that this build tag is only needed because tests for the code below couldn't be compiled with older Go version?

Reply:
That, and that with earlier go versions there's no use for these methods.

Reply:
1. They do have use because there was standalone implementation of `errors` package for older versions of Go: https://pkg.go.dev/golang.org/x/xerrors
2. There is no harm in their existence

Although, I don't have a strong point on this.

Reply:
Actually, I based this on a previous PR and did not give it much thought.
If I remove this condition, would it be OK not to add separate tests for older go versions?

Reply:
> would it be OK not to add separate tests for older go versions?

I think so.

Reply:
Done.